testing/mapnik/APKBUILD

```diff
@@ -0,0 +1,44 @@
+# Contributor: Markus Juenemann <markus@juenemann.net>
+# Maintainer: Markus Juenemann <markus@juenemann.net>
+pkgname=mapnik
+pkgver=v3.0.12
```

Comment:
Generally we don't want 'v' in the version number. Could we remove it here, and add it elsewhere? Or introduce some _pkgver=v$pkgver and use _pkgver elsewhere.